definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -264,9 +264,9 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateCitizenClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ResetPin", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "IssueGeneralLetter", "UserRole": "caseworker-cmc", "CRUD": "R"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CaseProceedsInCaseman", "UserRole": "caseworker-cmc", "CRUD": "R"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CaseProceedsInCaseman", "UserRole": "caseworker-cmc-judge", "CRUD": "R"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CaseProceedsInCaseman", "UserRole": "caseworker-cmc-legaladvisor", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CaseProceedsInCaseman", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CaseProceedsInCaseman", "UserRole": "caseworker-cmc-judge", "CRUD": "CR"},
```

Comment:
Will the judge and legal advisor do caseProceedinCaseman? Can you please confirm

Reply:
@SanchitaMoj Judge, Legal Advisor and Caseworker can all start this event.